src/NzbDrone.Core/Download/TorrentClientBase.cs

```diff
@@ -149,7 +149,9 @@ private string DownloadFromWebUrl(RemoteEpisode remoteEpisode, string torrentUrl
                             return DownloadFromMagnetUrl(remoteEpisode, locationHeader);
                         }
 
-                        return DownloadFromWebUrl(remoteEpisode, locationHeader);
+                        request.Url += new HttpUri(locationHeader);
```

Comment:
Is this going to work correctly with both relative and absolute location headers?